### DISCUSSION TITLE
Need more information about where's the master and volume running

### DISCUSSION THREAD

**Discussion by neo3matrix** [seaweedfs/seaweedfs-operator#42] [General]
Hello all, 
Not sure if this is "an issue" but wanted to ask the following:

I am newbie to seaweedfs overall. So far, I have been setting up seaweedfs using docker and used to run master and volume containers separately. I also have seen[ this example](https://github.com/chrislusf/seaweedfs/wiki/Deployment-to-Kubernetes-and-Minikube) where they run seaweedfs master+ volume on Minikube. 

But in case of seaweeds-operator, when I deployed it on my local k8s cluster, I could see only 1 pod is running "seaweedfs-operator-controller-manager" which contains 2 containers - 1 is manager & 1 is kube-rbac-proxy. 
The documentation says "Automatically deploy a SeaweedFS cluster with 3 masters, N volume servers, and M filers with customizable filer store managed by other operators."

I would like to know where/how can I see the 3 masters , N volumes running in this setup? 
Any guide, documentation or something of that sort would really help.
Please help.

**Comment by neo3matrix** [comment]
@chrislusf Please help.

**Reply by chrislusf** [reply]
how did you deploy it?

**Reply by neo3matrix** [reply]
I have a k8s cluster with 1 master & 2 worker nodes.

I Just followed the README instructions as follows:
1. git clone https://github.com/seaweedfs/seaweedfs-operator --depth=1
2. In my first attempt, I didn't want to use WEBHOOKS as I was just playing around - so didn't do ENABLE_WEBHOOKS & cert-manager steps.
3. make deploy 

In my 2nd attempt, on the same k8s cluster, I did everything:
1. git clone https://github.com/seaweedfs/seaweedfs-operator --depth=1
2. Installed cert-manager using helm chart.
3. ENABLE_WEBHOOKS=true in config/manager/manager.yaml
4. make deploy

In any of those approaches, I could only see only one pod "seaweedfs-operator-controller-manager" which contains 2 containers - one is manager & another one is kube-rbac-proxy.
I tried to exec into those containers to check if they are running master or volume services but couldn't run any commands on them like ps, top, ls, etc.

Am I missing something?

**Reply by neo3matrix** [reply]
@chrislusf  Any update/pointers?

**Reply by chrislusf** [reply]
related to https://github.com/seaweedfs/seaweedfs-operator/issues/39
the recent kubernetes seem have some changes.

@thiscantbeserious is working on this.

**Reply by thiscantbeserious** [reply]
@neo3matrix yes - that's the fault of the readme :) - all you did right now is deploy the operator itself.

Now you need to configurate it - I'll push the readme changes to help understanding that better I guess I can do that now ...

Currently getting more firm  on the basics and fixing a bug within cert-manager before I continue on the rebase.

**Reply by chrislusf** [reply]
@neo3matrix PR is merged. Thanks a lot!!!
